ruby/build_release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This file should be executed with ruby
+
+set -ex
+
+WORKING_DIR=$(dirname $(readlink $BASH_SOURCE))
```

Comment:
Does this dump the results in the source tree or in the sandbox?  If it's the former this seems very un-bazel-like

Reply:
It dumps in the source tree, but I think that's just how `sh_binary` works. I don't think it's a problem for release targets since we will always start with a clean tree and dump everything else.

Reply:
That's not how `sh_binary` is *supposed* to work IIUC.  This is non-hermetic and could cause issues for us during development/testing.  It would be much better to keep everything inside the Bazel sandbox, as it forces explicit dependency declaration and keeps the source tree isolated.  I could also see this playing badly with Bazel caching, since it's unaware of some of our dependencies

Reply:
As mentioned above, results are no longer dumped in the sandbox.